Lib/bz2.py

```diff
@@ -38,7 +36,7 @@ class BZ2File(_compression.BaseStream):
     returned as bytes, and data to be written should be given as bytes.
     """
 
-    def __init__(self, filename, mode="r", buffering=_sentinel, compresslevel=9):
+    def __init__(self, filename, mode="r", compresslevel=9):
```

Comment:
When remove a parameter, make all following parameters keyword-only. This will help to catch errors like `BZ2File(filename, 'w', 0)`. Please add also a test for such example.

Reply:
Good idea: done